### ISSUE TITLE
Unclear how to add Tinkers Complement Sledgehammer to whitelist

### ISSUE THREAD

**Issue by Garnin** [wendall911/TinkerSurvival#39]
Hey I love Tinkers Survival but I also want to be able to use the Ex Nihilo style hammer from Tinker's Complement. I keep getting a message saying that I need to use a pick on stone and cobblestone and I can't find an explanation anywhere of how to make it consider the sledgehammer a valid tool in this situation.

UPDATE: If I whitelist tcomplement in the mods whitelist then I can use the hammer on sand and gravel but not on anything that I would use a pick on like stone or ores or anything. 

I'm sorry if this should be easy but I've not been able to find anything in my search. Would a config line for easy  #compatibility be too much to request?

**Comment by BobbyGBs** [comment]
I second this. I've tried adding tcomplement to the mod tool white list and I've tried adding 
pickaxe-tcomplement:sledge_hammer
to tool white list and neither have worked. Tinkers survival just refuses to accept that the hammer is also a pickaxe. :/ 
please please please fix.

**Comment by D0CTOR-ZED** [comment]
I just wanted to say that adding some sort of tool type override would help not only this issue, but another mod compatibility issue as well (building gadget exchanging gadget can't affect stone, suspected cause is it's not a pick).  This could be a tool type override (allow-to-be-a-pick: hammer, etc.) or a can be any tool override (doesnt-verify-tool-type: hammer, etc.)

**Comment by happysmash27** [comment]
As far as I can tell, it is, in fact, impossible to add the sledgehammer to the whitelist and have it work, even if formatted properly, due to the harvest level of it being -1 and TinkerSurvival viewing that, as the tool not being able to harvest the block. 

In the process of figuring out that this was the issue, though, (I had to look into the source code), I made a pull request to fix it, for both Ex Nihilo's own Tinker's Construct hammer (I only realised it had one after viewing its source code, then testing it without Tinkers' Compliment) and Tinkers' Complement's version of it. So, with a few small code changes, it should be able to work well by default!
